code/__DEFINES/colors.dm

```diff
@@ -39,14 +39,12 @@
 #define COLOR_ETHIOPIA_RED "#DA121A"
 #define COLOR_UNION_JACK_RED "#C8102E"
 #define COLOR_MEDIUM_DARK_RED "#CC0000"
-#define COLOR_PINK_RED "EF3340"
 #define COLOR_SYNDIE_RED "#F10303"
 #define COLOR_SYNDIE_RED_HEAD "#760500"
 #define COLOR_MOSTLY_PURE_RED "#FF3300"
 #define COLOR_DARK_RED "#A50824"
 #define COLOR_RED_LIGHT "#FF3333"
 #define COLOR_FIRE_LIGHT_RED "#B61C1C"
-#define COLOR_MAROON "#800000"
```

Comment:
would prefer we kept color defines because there is some use to having these in the future for people who want to go "oooh i don't want that level of red..." and having to make up some other completely new hex code and name